js/multiFileUpload.js

```diff
@@ -1,351 +1,390 @@
-(function ($)
+(function ($) {
+
+  "use strict";
+
+  Qubit.multiFileUpload = Qubit.multiFileUpload || {};
```

Comment:
I doubt this will work if `Qubit.multiFileUpload` doesn't exist and it's populated with and empty object, so maybe we can remove this line.

Reply:
Removed.

Reply:
Leaving in as it seems necessary. If removed then Qubit.multiFileUpload is unknown to the js function.